concent_api/core/payments/validation.py

```diff
@@ -0,0 +1,20 @@
+from core.exceptions import PaymentTimestampError
+from core.payments.backends.sci_backend import get_list_of_payments
+from core.payments.backends.sci_backend import TransactionType
+
+
+def validate_that_last_closure_time_is_older_than_oldest_payment(  # pylint: disable=inconsistent-return-statements
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    youngest_payment_ts: int,
+) -> None:
+    forced_payment_event_list = get_list_of_payments(
+        requestor_eth_address=requestor_eth_address,
+        provider_eth_address=provider_eth_address,
+        payment_ts=youngest_payment_ts,
+        transaction_type=TransactionType.FORCE,
+    )
+    if forced_payment_event_list == []:
```

Comment:
please change it to:
```python
if forced_payment_event_list != []:
 if youngest_payment_ts < max([event.closure_time for event in forced_payment_event_list]):
        raise PaymentTimestampError
```
 you will not have to add unnecessary `return` statements and disable pylint :>

Reply:
And also in @dybi version those two conditions can be combined into one.

Reply:
done

Reply:
@rwrzesien why do you insist on combining the conditions? Isn't it readable the way it is? If we combine those two, the line will become really long and/or will require weird formatting